packages/redux/src/settings/actions/factories/fetchAccountSettingFactory.ts

```diff
@@ -0,0 +1,52 @@
+import * as actionTypes from '../../actionTypes.js';
+import {
+  type AccountSetting,
+  type Config,
+  type GetAccountSetting,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import type { Dispatch } from 'redux';
+import type { FetchAccountSettingAction } from '../../types/index.js';
+
+/**
+ * Creates a thunk factory configured with the specified client to fetch all
+ * settings.
```

Comment:
`Creates a thunk factory configured with the specified client to fetch a setting by id.`

Reply:
Fixed it as well 👍